packages/api/src/constants.ts

```diff
@@ -4,6 +4,8 @@
 
 export const ConfigFolderName = "fx";
 export const AppPackageFolderName = "appPackage";
+export const ArchiveFolderName = ".archive";
+export const ArchiveLogFileName = ".archive.log";
```

Comment:
What is this log file for? Seems not used in this PR.

Reply:
Sorry, the file is used by two prs at the same time. The log file used in the archive step.